packages/angular/projects/ui-angular/src/index.ts

```diff
@@ -20,6 +20,7 @@ export * from './lib/primitives';
 export * from './lib/primitives-poc/button';
 export * from './lib/primitives-poc/view';
 export * from './lib/primitives-poc/text-field';
+export * from './lib/primitives-poc/heading';
```

Comment:
nit: shall we sort the exports ascending?

Reply:
https://github.com/aws-amplify/amplify-ui/pull/2433/commits/25ef7d1da56542c2030145f96640dd45dcc59708